### ISSUE TITLE
Custom information table

### ISSUE THREAD

**Issue by djtecha** [librenms/librenms#2874]
It would be pretty neat if we could create custom key:value pairs to a tab for a device. Either using the API (seems easier to be generic) or custom Mibs for servers. I'd like to completely move away from racktables and keep all the inventory in librenms, so something that would give the flexibility of something like the "notes" section but a little more organized so we could do reporting would be sweet.

IE: Curl PUT "KEY:VALUE"

A table on a "Info" tab

--------------------------------------------------------
| KEY | VALUE 
| KEY | VALUE
--------------------------------------------------------

**Comment by laf** [comment]
If I understand what you mean then we can do this with:

http://docs.librenms.org/Extensions/Component/

**Comment by djtecha** [comment]
Yes, that would work. Though I'm confused as to how one actually updates those tables. Looks like there's only a GET and not a PUT for the API.
